docs/_templates/partials/head.njk

```diff
@@ -1,31 +1,14 @@
 <head>
-  <title>{% if page.title %}{{page.title}} - {% endif %}{{site.title}}</title>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>{% if page.title %}{{page.title}} - {% endif %}{{site.title}}</title>
   <base href="{{site.baseHref}}">
 
   <link rel="shortcut icon" href="images/favicon.png">
 
-  <!-- Font Awesome Brand Icons -->
-  <link
-    crossorigin="anonymous"
-    href="https://use.fontawesome.com/releases/v5.2.0/css/brands.css"
-    integrity="sha384-nT8r1Kzllf71iZl81CdFzObMsaLOhqBU1JD2+XoAALbdtWaXDOlWOZTR4v1ktjPE"
-    rel="stylesheet"
-  />
-  <link
-    crossorigin="anonymous"
-    href="https://use.fontawesome.com/releases/v5.2.0/css/fontawesome.css"
-    integrity="sha384-HbmWTHay9psM8qyzEKPc8odH4DsOuzdejtnr+OFtDmOcIVnhgReQ4GZBH7uwcjf6"
-    rel="stylesheet"
-  />
```

Comment:
Brought in SVG icons from fontawesome, instead of loading more assets than necessary.